src/misc/index.js

```diff
@@ -11,3 +11,4 @@ export { DefaultAvatar } from './DefaultAvatar';
 export { FadingTooltip } from './FadingTooltip';
 export { TooltipInfo } from './TooltipInfo';
 export { PermissionsGate } from './PermissionsGate';
+export { TableFilterRow } from './TableFilterRow';
```

Comment:
The component `TableFilterRow` does not provide enough value to be exported as an officially supported component of our components library, it is only used by the `Table` component. Could you remove this export, and move the `TableFilterRow` in a `components` folder, in `src/inputs/Table` directory, please ?